app/views/play/level/ControlBarView.coffee

```diff
@@ -75,7 +75,10 @@ module.exports = class ControlBarView extends CocoView
     c.spectateGame = @spectateGame
     c.observing = @observing
     @homeViewArgs = [{supermodel: if @hasReceivedMemoryWarning then null else @supermodel}]
-    if @level.get('type', true) in ['ladder', 'ladder-tutorial', 'hero-ladder', 'course-ladder']
+    if me.isTeacher()
+      @homeLink = "/teachers/courses"
+      @homeViewClass = "views/courses/TeacherCoursesView"
```

Comment:
Style: prefer single quotes in .coffee files not using interpolation or containing other single quotes.